### PR TITLE
Change healthz port

### PR DESCRIPTION
**Pull request by vendrov**
According to this issue the port was changed:
https://github.com/kubernetes/kubernetes/issues/10003

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1826)

### CODE REVIEW

File: docs/admin/kubelet.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ DynamicVolumeProvisioning=true|false (ALPHA - default=true)
       --google-json-key string                         The Google Cloud Platform Service Account JSON Key to use for authentication.
       --hairpin-mode string                            How should the kubelet setup hairpin NAT. This allows endpoints of a Service to loadbalance back to themselves if they should try to access their own Service. Valid values are "promiscuous-bridge", "hairpin-veth" and "none". (default "promiscuous-bridge")
       --healthz-bind-address value                     The IP address for the healthz server to serve on, defaulting to 127.0.0.1 (set to 0.0.0.0 for all interfaces) (default 127.0.0.1)
-      --healthz-port value                             The port of the localhost healthz endpoint (default 10248)
+      --healthz-port value                             (Deprected) The port of the localhost healthz endpoint (default 10248)
```
Comment (jaredbhatti) [inline docs/admin/kubelet.md]: Deprected -> Deprecated?

**Comment by keglevich3** [comment]
typo fixed

**Comment by devin-donnelly** [comment]
@vendrov This has a merge conflict that needs fixing.

**Comment by chenopis** [comment]
@vendrov If you can resolve these conflicts, we can merge this.

**Comment by chenopis** [comment]
Moving this to PR #2414

**Pull request closed without merging**